--- configure.ac    2021-04-22 19:32:00.000000000 +0100
+++ configure.ac.modif    2021-04-22 19:09:01.000000000 +0100
@@ -134,8 +134,8 @@
 AM_CONDITIONAL([WITH_NET], [test "x$with_net" = "xyes"])
 AM_CONDITIONAL([WITH_LOGDB], [test "x$with_tools" = "xyes" -o "x$with_wallet" = "xyes"])

-ac_configure_args="${ac_configure_args} --enable-module-recovery"
-AC_CONFIG_SUBDIRS([src/secp256k1])
+#ac_configure_args="${ac_configure_args} --enable-module-recovery"
+#AC_CONFIG_SUBDIRS([src/secp256k1])

 dnl make sure nothing new is exported so that we don't break the cache
 PKGCONFIG_PATH_TEMP="$PKG_CONFIG_PATH"
